modules/howtos/pages/encrypting-using-sdk.adoc

```diff
@@ -1,15 +1,175 @@
 = Field Level Encryption from the SDK
-:description: The Field Level Encryption library enables encryption and decryption of JSON fields, to support FIPS-140-2 compliance.
+:description: A practical guide for getting started with Field-Level Encryption, showing how to encrypt and decrypt JSON fields using the Python SDK.
 :page-topic-type: howto
 :page-edition: Enterprise Edition
-:page-status: Developer Preview
-// // :page-aliases: ROOT:encrypting-using-sdk.adoc
+//:page-aliases: ROOT:encrypting-using-sdk.adoc

```

Comment:
Think we need to un-comment this @RichardSmedley?

Reply:
done